virocon/plotting.py

```diff
@@ -293,8 +317,138 @@ def plot_dependence_functions(model, semantics=None, par_rename={}, axes=None):
     return axes
 
 
+def plot_histograms(model, sample, semantics=None, plot_pdf=True):
+    """
+    Plot the histograms of data along with the density of the models distributions.
+
+    Parameters
+    ----------
+    model : MultivariateModel
+        The fit model.
+    sample : ndarray
+        The data that was used to fit the model.
+    semantics: dict, optional
+        The description of the model. If None (the default) generic semantics will be used.
+    plot_pdf: boolean, optional
+        Whether the fitted probability density should be plotted. Defaults to True.
+    Returns
+    -------
+    The used matplotlib axes objects.
+
+    """
+    sample = np.asarray(sample)
+    n_dim = model.n_dim
+
+    if semantics is None:
+        semantics = get_default_semantics(n_dim)
+
+    figures = []
+    axes_list = []
+
+    for dim in range(n_dim):
+
+        x_name = semantics["names"][dim]
+        x_symbol = semantics["symbols"][dim]
+        x_unit = semantics["units"][dim]
+        x_label = f"{x_name}," + r" $\it{" + f"{x_symbol}" + r"}$" + f" ({x_unit})"
+
+        if model.conditional_on[dim] is None:
+            # unconditional
+            data = sample[:, dim]
+            dist = model.distributions[dim]
+            x = np.linspace(np.min(data), np.max(data))
+            fig, ax = plt.subplots()
+            ax.hist(
+                data,
+                bins="doane",
+                density=True,
+                color="#000000",
+                histtype="stepfilled",
+                alpha=0.2,
+            )
+            if plot_pdf:
+                density = dist.pdf(x)
+                ax.plot(
+                    x,
+                    density,
+                    color="#004488",
+                )
+            ax.set_xlabel(x_label)
+            ax.set_ylabel("probability density")
```

Comment:
Capitalize "Probability" as we seem to follow the convention to capitalize the first word in our axis label. https://virocon.readthedocs.io/en/latest/detailed_examples.html#year-environmental-contour-hs-tz-dnvgl-model